README.md

```diff
@@ -7,11 +7,15 @@ tagging versions.
 
 Clone the [BitBox02 firmware repo](https://github.com/digitalbitbox/bitbox02-firmware):
 
+Make sure you have `protoc` and
+[protoc-gen-go](https://developers.google.com/protocol-buffers/docs/reference/go-generated)
+installed:
+
 `git clone https://github.com/digitalbitbox/bitbox02-firmware.git`
 
 ```sh
 rm -rf api/firmware/messages/*
```

Comment:
This removes also `generate.sh`. I would suggest to use `rm -rf api/firmware/messages/*.proto` and `rm -rf api/firmware/messages/*.go` or to move `generate.sh` in another folder

Reply:
great catch! fixed